From 518a832f7f95bc2f0eb013ebe3e4ae1c21b5812f Mon Sep 17 00:00:00 2001
From: Hugo Osvaldo Barrera <hugo@whynothugo.nl>
Date: Sat, 30 Aug 2025 14:39:55 +0200
Subject: [PATCH] Fix mixup in test

test_fp_mode called get_fpexc. This failed in architectures which
support get_fpexc but not get_fp_mode.
---
 test_prctl.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test_prctl.py b/test_prctl.py
index 5b9aa1c..6c6f59d 100644
--- a/test_prctl.py
+++ b/test_prctl.py
@@ -172,8 +172,8 @@ class PrctlTest(unittest.TestCase):
             self.assertEqual(prctl.get_fp_mode(), prctl.FP_MODE_FRE)
             self.assertRaises(ValueError, prctl.set_fp_mode, 999)
         else:
-            self.assertRaises(OSError, prctl.get_fpexc)
-            self.assertRaises(OSError, prctl.set_fpexc)
+            self.assertRaises(OSError, prctl.get_fp_mode)
+            self.assertRaises(ValueError, prctl.set_fp_mode)
 
     @require('set_io_flusher')
     def test_io_flusher(self):
-- 
2.51.0

